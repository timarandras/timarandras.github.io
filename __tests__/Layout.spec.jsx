import { render } from '@testing-library/react';
import React from 'react';

import Layout from '../src/components/Layout';

describe('Layout', () => {
  test('Matches snapshot', () => {
    const { container } = render(
      <Layout>
        <div>content</div>
      </Layout>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  test('Has correct content', () => {
    const { getByText } = render(<Layout>content</Layout>);

    expect(getByText('content')).toHaveTextContent('content');
  });

  it('Has maxWidth set to large', () => {
    const { getByText } = render(<Layout>content</Layout>);

    expect(getByText('content').closest('div')).toHaveClass('MuiContainer-maxWidthLg');
  });
});
