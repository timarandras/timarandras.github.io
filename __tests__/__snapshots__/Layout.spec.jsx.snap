// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Matches snapshot 1`] = `
<div
  class="MuiContainer-root MuiContainer-maxWidthLg"
>
  <div>
    content
  </div>
</div>
`;
